### PR TITLE
Bookstore initialize

### PR DESCRIPTION
**Pull request by Kccendana**
### **In this Pull Request, the following was done:**

- Setup a new React application
- Created a directory for reusable components
- Installed React Router
- Setup your router
- Add 2 routes (home and categories)
- Created components inside of the /components directory

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <>
+    <Nav />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="categories" element={<Categories />} />
```
Comment (usorfaitheloho) [inline src/App.js]: Kindly note that the routes should be `/categories` and not 'categories' as suggested in the project requirement , so as to make your app work better 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Kccendana  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> *Use React Router. ✔️
> * Use React components ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Kccendana) [reply src/App.js]: Thank you so much. I will do that right away..

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @Kccendana  :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- App runs well and functions as expected ✅ 
- App was created with `Create-React-App`  :heavy_check_mark: 
- Beautiful UI styles :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**